pyqtgraph/opengl/items/GLMeshItem.py

```diff
@@ -181,7 +181,7 @@ def paint(self):
                     if self.colors is None:
                         color = self.opts['color']
                         if isinstance(color, QtGui.QColor):
-                            glColor4f(*fn.glColor(color))
+                            glColor4f(*color.getRgbF())
```

Comment:
As the code checker points out, it seems you have successfully eliminated the dependency on pg.functions !

Reply:
There are inconsistencies with the opengl items' APIs. Some take in float colors, some take in an argument for mkColor.